AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Features/Tasks/TaskStatusChangeHandlerTests.cs

```diff
@@ -96,41 +96,57 @@ protected override void LoadTestData()
         [Fact]
         public async Task VolunteerAssignedTask()
         {
+            var dateTime = DateTime.UtcNow;
             var task = Context.Tasks.First();
             var user = Context.Users.First();
             var command = new TaskStatusChangeCommandAsync
             {
                 TaskId = task.Id,
                 UserId = user.Id,
-                TaskStatus = TaskStatus.Assigned
+                TaskStatus = TaskStatus.Assigned,
+                TaskStatusDescription = $"Assign {task.Name} to {user.UserName}"
             };
             await handler.Handle(command);
 
             var taskSignup = Context.TaskSignups.First();
             // Verify that the handle method publishes a notification that the task status changed
             mediator.Verify(b => b.PublishAsync(It.Is<TaskSignupStatusChanged>(notifyCommand => notifyCommand.SignupId == taskSignup.Id)), Times.Once());
-            // Make sure the task signup has the expected status
-            taskSignup.Status.ShouldBe(TaskStatus.Assigned.ToString());
+            // Make sure the task signup properties are the expected values
+            // based on the command.
+            taskSignup.Status.ShouldBe(command.TaskStatus.ToString());
+            taskSignup.Task.Id.ShouldBe(command.TaskId);
+            taskSignup.User.Id.ShouldBe(command.UserId);
+            taskSignup.StatusDescription.ShouldBe(command.TaskStatusDescription);
+            // the datetimes should be within 1 second of each other
+            taskSignup.StatusDateTimeUtc.ShouldBe(dateTime, TimeSpan.FromSeconds(1));

```

Comment:
take a look at how DateTime is tested in `UpdateMyTasksHandlerAsync` and `UpdateMyTasksHandlerAsyncShould`.  We use a func to pass in what we need for the unit test, and at runtime, the func resolves itself. Up to you if you want to test dates this way. This is pretty much how I do it, but there is not "standard" way for the code for AllReady